src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
+
+export interface TodoWithUser extends Todo {
+  user: User | undefined,
```

Comment:
nit: Consider using `user?: User` instead of `user: User | undefined` for an optional property. 🚀